### DISCUSSION TITLE
How can I get the Contact detail types by using DPF.?

### DISCUSSION THREAD

**Discussion by Naveen-Kumar-B** [ansys/pydpf-post#588] [Q&A]
I need to get the contact details (types) by reading the result file only using DPF.

@PProfizi pls help me out.

**Comment by pmaroneh** [comment]
This was also asked here: https://discuss.ansys.com/discussion/2994/how-can-i-get-the-contact-detail-types-by-using-dpf#latest

**Reply by Naveen-Kumar-B** [reply]
Yes @pmaroneh , I just posted over their..

**Comment by PProfizi** [comment]
Hi @B-Naveen-Kumar,
I'll need more detail on what you need exactly.
Are you looking for the MAPDL contact element type number?

**Reply by Naveen-Kumar-B** [reply]
hi @PProfizi 

I need to get the contact details like contact pressure, contact force, contact stress etc through DPF.

Not about APDL ETYPE.

**Reply by pmaroneh** [reply]
One way to go is to grab the SMISC/NMISC values of the APDL elements. There are some examples here: 
https://discuss.ansys.com/discussion/comment/3501#Comment_3501?utm_source=community-search&utm_medium=organic-search&utm_term=contact+dpf
https://discuss.ansys.com/discussion/comment/2579#Comment_2579?utm_source=community-search&utm_medium=organic-search&utm_term=contact+dpf
https://discuss.ansys.com/discussion/comment/2575#Comment_2575?utm_source=community-search&utm_medium=organic-search&utm_term=contact+dpf

**Reply by Naveen-Kumar-B** [reply]
@pmaroneh Thanks for sharing this information.
